yabt/builders/python_test.py

```diff
@@ -47,3 +47,28 @@ def test_python_tester_fail(basic_conf):
     populate_targets_graph(build_context, basic_conf)
     with pytest.raises(SystemExit):
         build_context.build_graph(run_tests=True)
+
+
+@slow
+@pytest.mark.usefixtures('in_tests_project')
+def test_python_tester_fail_with_retry(basic_conf):
+    build_context = BuildContext(basic_conf)
+    basic_conf.targets = ['hello_pytest:greet-failing-test']
+    populate_targets_graph(build_context, basic_conf)
+    target = build_context.targets[basic_conf.targets[0]]
```

Comment:
clearer if you get the target by name, don't you think?

Reply:
See "latent-retries" branch where I learn what's "target.name" and what's "target"
I'll fix it in next push (to this branch)

Reply:
Oh wait, I misread, you want me to use the string above to get the target?
I prefer taking it into a variable and use in both places.

Reply:
ok, so at least just move the string on L56 to a variable (`target_name = '...'`) and use the variable in both places

Reply:
Done